source/includes/info.md

```diff
@@ -124,12 +124,12 @@ POST /openmrs/ws/rest/v1/order
 {"t": "testorder", /*... and other properties */}
 ```
 
-- If you GET a resource that has subtypes, each result will be of one of those subtypes,
+- If you GET a resource that has subtypes, each result will be one of those subtypes,
   which you can see by looking at the special property of each result.
 
-- You may query for only a certain subtype of a resource by providing a t query parameter.
+- You may query for only a certain subtype of a resource by providing at the query parameter.
```

Comment:
this is wrong revert back

Reply:
yeah sure @Akayeshmantha  this is wrong I got confused !! I will revert back